messagingpush/src/main/java/io/customer/messagingpush/ModuleMessagingPushFCM.kt

```diff
@@ -26,18 +27,17 @@ internal constructor(
         overrideDiGraph = null
     )
 
-    private val customerIO: CustomerIOInstance
-        get() = overrideCustomerIO ?: CustomerIO.instance()
-    private val diGraph: CustomerIOComponent
-        get() = overrideDiGraph ?: CustomerIO.instance().diGraph
-    private val fcmTokenProvider by lazy { diGraph.fcmTokenProvider }
+    private val diGraph: SDKComponent
+        get() = SDKComponent
+
+    private val fcmTokenProvider by lazy { diGraph.androidSDKComponent?.fcmTokenProvider }
```

Comment:
We do need context here, regarding assuming we will always have context here, we would need to add it to constructor, but its a public facing constructor so we can't update it without providing a default which will have to be nullable. 
So this would be more work to make sure public interface doesn't break and we can have this has non nullable.